### PR TITLE
Play Video

### PR DESCRIPTION
**Pull request by loemhuolay**
# Play Video 
- Template for play video

For route action <`get 'videos/play'`>
For play video <`http://localhost:3000/movies/show`>
For style css <`app/assets/stylesheets/video.scss`>

### CODE REVIEW

File: pj_movie/config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  get 'videos/new'
```
Comment (N310) [inline pj_movie/config/routes.rb]: I think `videos#new` don't need for user page.

File: pj_movie/test/controllers/videos_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```
Comment (N310) [inline pj_movie/test/controllers/videos_controller_test.rb]: In this project, we use Rspec, don't use minitest.
So you should change to rspec test.

File: pj_movie/app/views/videos/play.html.erb [hunk 3]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+  </head>
+  <body>
```
Comment (N310) [inline pj_movie/app/views/videos/play.html.erb]: this code is unneeded.

File: pj_movie/app/views/videos/play.html.erb [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+            <div class="video_items_info_title">
+              This is my video title.
+            </div>
+            <p>140 views</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (N310) [inline pj_movie/app/views/videos/play.html.erb]: this code is unneeded.

File: pj_movie/app/assets/stylesheets/videos.scss [hunk 5]
```diff
@@ -0,0 +1,105 @@
+.container{
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+}
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/videos.scss]: this `.container` is common style in all page ? or it is needed in page of video play ?

File: pj_movie/app/assets/stylesheets/videos.scss [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+.video{
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/videos.scss]: I think it is better that you create parent class, and the parent class include all style which used in video play .

File: pj_movie/app/assets/stylesheets/videos.scss [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+    display: flex;
+    flex-direction: column;
+  }
+  .cover_video_side{
+    width: 90%;
+  }
+  .video{
+    width: 100%;
+    display: flex;
+    // justify-content: center;
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/videos.scss]: you should remove unused style.

File: pj_movie/app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>PjMovie</title>
+    <title>Movie App</title>
```
Comment (N310) [inline pj_movie/app/views/layouts/application.html.erb]: Do you need this change in the branch ?

File: pj_movie/config/routes.rb [hunk 9]
```diff
@@ -1,3 +1,8 @@
...
+
+  get 'videos/play'
```
Comment (N310) [inline pj_movie/config/routes.rb]: you should use `resources`, and change play action change to show action.

File: pj_movie/app/views/layouts/application.html.erb (hunk 8 above)
Comment (loemhuolay) [reply pj_movie/app/views/layouts/application.html.erb]: Yes

File: pj_movie/config/routes.rb [hunk 10]
```diff
@@ -1,8 +1,7 @@
...
-
-
-  root 'static_pages#home'
-  get 'static_pages/home'
-  get 'static_pages/help'
+  resources :movies do
+    collection do
+      get 'play'
+    end
+  end
```
Comment (N310) [inline pj_movie/config/routes.rb]: you had better that `play action` change to `show action`.

File: pj_movie/config/routes.rb [hunk 11]
```diff
@@ -1,4 +1,10 @@
 Rails.application.routes.draw do
+  resources :movies do
+    collection do
+      get 'show'
+    end
+  end
```
Comment (N310) [inline pj_movie/config/routes.rb]: you don't need to use `collection`.
And you should move to under the root.

File: pj_movie/config/routes.rb [hunk 12]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :movies, only: [:show]
```
Comment (N310) [inline pj_movie/config/routes.rb]: You should move to under the root.

**Pull request merged**